### DISCUSSION TITLE
Intel announces successor: Phantom Canyon

### DISCUSSION THREAD

**Discussion by osy** [osy86/HaC-Mini#557] [General]
https://simplynuc.com/phantom-canyon/

Intel Core i7-1165G7 + discrete NVIDIA RTX 2060 6GB starting at $1349 USD

**Comment by osy** [comment]
Thoughts:

* Price is too high. Might as well get a gaming laptop at this point. Or the GPD Win 3 which has the same CPU for $799 and a screen at the same dimensions.
* NVIDIA 2060 instead of 3060? Already outdated.
* Less ports overall
* No hackintosh drivers exist for NVIDIA GPU so a full hackintosh project is not feasible.

It’s a pass for me. I’ll probably get the next Apple Silicon Mac when it comes out.

**Comment by Vitoli-o** [comment]
Thanks for this @osy! I was not aware this was launching so soon... 

I think I wouldn't disregard it purely because it cannot be turned into a full hackintosh. We don't yet know whether the integrated graphics can be exploited to pass video other than the RTX 2060. If it does, it's going to be a very good buy, particularly if you're not using it for gaming, but rather coding or photo editing/web design/office work.

Let's wait and see the reviews!

**Comment by osy** [comment]
Right but at that price, if I’m not using the GPU, I would rather get a GPD Win 3 + dock which has similar dimensions.

**Comment by Vitoli-o** [comment]
You're perfectly right on the dGPU point.

This is more suitable for those (like me) who like to dual boot, particularly for gaming.

Realistically, you cannot play GTA or Cyberpunk under macOS with all the instabilities that come with Big Sur and the HaCMini... (at least I don't want to put up with it!).

I'm not sure whether an Xe Graphics is compatible with Big Sur, as I have not seen any MacBooks yet with this iGPU nor I'm aware of any benchmarks with Geekbench 5 under Big Sur to be compared with our dGPU.

This is definitely exciting, and worth considering IMO!

**Comment by KrisCris** [comment]
Totally agree. Waiting for the next-gen apple silicon mac mini would be a better choice for macOS users in my opinion.

Considering its performance and price, build a full desktop with the latest specs is a wiser choice for those who want gaming.

**Comment by Medizinmann-MD** [comment]
AFAIK Xe Graphics right now isn't compatible with any version auf MacOS - correct me if I am wrong.

**Comment by Vitoli-o** [comment]
> AFAIK Xe Graphics right now isn't compatible with any version auf MacOS - correct me if I am wrong.

Good point!

❗ Here is the product specification: https://www.intel.com/content/dam/support/us/en/documents/intel-nuc/NUC11PH_TechProdSpec.pdf

From my understanding, both USB-C ports can output graphics with the iGPU (UHD or Xe?), whereas HDMI/mDP can output through RTX 2060 💡 

Also, you cannot mount 2 x SATA SSDs, which is my current configuration on the Hades Canyon 🤔

**Comment by osy** [comment]
Also: I would be surprised if Apple adds Xe drivers in the future considering they’re moving to Apple Silicon.
